### PR TITLE
# setting up tailwindcss && adding the Pr template

### PR DESCRIPTION
**Pull request by Jaman-dedy**
#### Description of changes made

> This PR adds configuration to add linting checks to the codebase.
**This PR will add** : 
-  The tailwindcss setup
- The PR template

#### Type of change

> **Please select the relevant options**

- [ ] New feature (non-breaking changes which add the new functionalities)
- [ ] Chore (non-breaking changes which fix an issue)
- [x] Documentation (added or improved the documentation)
- [ ] Breaking change (fix or a feature that would cause existing feature not to work as expected)
- [ ] This change requires a documentation update

#### How Has This Been Tested?

> **Please describe the tests that you ran to verify your changes**
- After cloning the [repo](https://github.com/kisanola/AlphaCodeFrontend.git)
- Check out this branch by `ch-set-up-tailwindcss`
- Run `yarn or npm install` 

- [ ] Unit
- [ ] Integration
- [ ] End to end

> **Provide the steps to manually test the functionalities implemented in this PR**

#### Checklist

- [x] My code follows the style guidelines of this project
- [x] I have ensured coding conventions for my code prior to submission
- [ ] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings
- [ ] I have added tests which prove the effectiveness of my work
- [ ] New and existing tests pass locally with my changes

### CODE REVIEW

File: PULL_REQUEST_TEMPLATE.md [hunk 1]
```diff
@@ -0,0 +1,45 @@
+#### Description of changes made
```
Comment (espoirMur) [inline PULL_REQUEST_TEMPLATE.md:1]: This is nice,...!!

Comment (espoirMur) [reply PULL_REQUEST_TEMPLATE.md:1]: Maybe you should add a paragraph, where we can put the link to the [gitKraten](https://www.gitkraken.com/) card.

File: src/App.tsx [hunk 2]
```diff
@@ -1,25 +1,35 @@
 import React from 'react';
-import logo from './logo.svg';
-import './App.css';
+import './styles/app.css';
 
 function App() {
```
Comment (espoirMur) [inline src/App.tsx:4]: shouldn't this be in ES6? :thinking_face:

**Review by espoirMur — COMMENTED** [review]
LGTM

File: package.json [hunk 3]
```diff
@@ -16,10 +16,12 @@
     "typescript": "~3.7.2"
   },
   "scripts": {
-    "start": "react-scripts start",
-    "build": "react-scripts build",
+    "start": "npm run watch:css && react-scripts start",
+    "build": "npm run build:css && react-scripts build",
```
Comment (gracelungu) [inline package.json:20]: @Jaman-dedy What is the purpose of the build and watch you have added?

**Review by gracelungu — COMMENTED** [review]
LGTM

**Review by espoirMur — APPROVED** [review]

**Pull request merged**